bridge/client/app/_services/api.service.ts

```diff
@@ -125,6 +126,24 @@ export class ApiService {
       .pipe(catchError(this.handleError<EventResult>('getEvaluationResults')));
   }
 
+  public sendApprovalEvent(approval: Trace, approve: boolean) {
+    let url = `${this._baseUrl}/v1/event`;
+    return this.http
+      .post<any>(url, {
+        "shkeptncontext": approval.shkeptncontext,
+        "type": EventTypes.APPROVAL_FINISHED,
+        "triggeredid": approval.id,
+        "source": "https://github.com/keptn/keptn/bridge#approval.finished",
+        "data": Object.assign(approval.data, {
+          "approval": {
+            "result": approve ? ApprovalStates.APPROVED : ApprovalStates.DECLINED,
+            "status": "succeeded"
```

Comment:
"succeeded" are there more status of the approval event? if yes then they should be in a enum.

Reply:
"succeeded" is actually a constant and should always be "succeeded" in the case that the Approval comes from the Bridge.